portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
-*/
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function getMessages() {
+  fetch('/data').then(response => response.json()).then((messages) => {
+    const messagesElement = document.getElementById('messages-container');
+    console.log(messages);
+    for (var i = 0; i < messages.length; i++) {
+        messagesElement.innerHTML += "<p>"+messages[i]+"</p><br>";
```

Comment:
yeah, I think you're stuck using innerHTML for this. But is appending appropriate here?  What happens if you click the button multiple times?